### DISCUSSION TITLE
how to switch off the monitor mode?

### DISCUSSION THREAD

**Discussion by ryxxastroat** [abraunegg/onedrive#1833] [Q&A] [answered]
Hello, I'm finding that files are automatically uploaded and downloaded. I assume it is in the monitor mode. How can I switch off the monitor mode and just synchronize manually whenever I want? Thanks!

**Comment by abraunegg** [comment, marked as answer]
@ryxxastroat 
> Hello, I'm finding that files are automatically uploaded and downloaded. I assume it is in the monitor mode. How can I switch off the monitor mode and just synchronize manually whenever I want?

This 100% sounds like Issue #1274 - and probably, when installing the client for Ubuntu you missed reading the known issues as documented here: https://github.com/abraunegg/onedrive/blob/master/docs/ubuntu-package-install.md#known-issues-with-installing-from-the-above-packages

The easiest way to stop this is:
1. Disable any 'onedrive' systemd service
2. Remove the link that the Ubuntu packages create
3. Reboot

If you have *not* installed from the OpenSuSE Build Service and *not* using at least version v2.4.15 of the application - you are using an unsupported and obsolete client version.

**Reply by ryxxastroat** [reply]
Thank you very much for the resolution! I did install the Ubuntu PPA package from yann1cks before. That's how the issue comes.
